src/components/NodeToolbar/NodeToolbar.js

```diff
@@ -271,6 +272,10 @@ class NodeToolbar extends React.Component<Props, State> {
     const reportButtonCaption = t`Report an Issue`;
     const placeName = placeNameFor(get(this.props, 'feature.properties'), this.state.category);
 
+    // translator: Button caption shown in the place toolbar
+    const openInMapsCaption = t`Open in Maps App`;
```

Comment:
It would be cool to know the exact app that will be opened when clicking the link (for non-Android). What do you think?

Reply:
So `Open in Apple Maps` etc.?

Reply:
Yep!